src/mutator.cc

```diff
@@ -678,20 +679,4 @@ std::string Mutator::MutateUtf8String(const std::string& value,
   return str;
 }
 
-bool Mutator::ApplyCustomMutations(protobuf::Message* message,
-                                   const protobuf::FieldDescriptor* field) {
-  auto itr = custom_mutations_.find(field);
-  if (itr == custom_mutations_.end())
-    return false;
-
-  // Randomly select one of the registered mutators. The default behavior is
-  // performed for index 0.
-  size_t field_index = GetRandomIndex(&random_, itr->second.size() + 1);
```

Comment:
We're losing the ability to register more than one mutator for a given field.
I can't think of a case where this would be useful off the top of my head, but something to consider.